client/src/components/ActivityShow.js

```diff
@@ -7,16 +8,15 @@ const ActivityShow = (props) => {
     description: "",
   });
 
+  const activityId = props.computedMatch.params.id;
   const getActivity = async () => {
-    const activityId = props.computedMatch.params.id;
     try {
       const response = await fetch(`/api/v1/activities/${activityId}`);
       if (!response.ok) {
         const errorMessage = `${response.status} (${response.statusText})`;
         const error = new Error(errorMessage);
         throw error;
       }
-
       const body = await response.json();
       setActivity(body.activity);
```

Comment:
since the reviews are on this same page, we should reduce/ remove the fetch for reviews in the show component, and instead query for the associated reviews from this activity API endpoint